docs/io/configuration/components/models/data/config_init_trad.yml

```diff
@@ -9,7 +9,7 @@ supernova:
   time_explosion: 16 day
   distance: 8.32 Mpc
 
-# standard atomic data base; get it from the tardis-refdata repository
+# standard atomic data base; get it from the tardis-regression repository
```

Comment:
Use the full repository name everywhere, `tardis-regression-data`

Reply:
sure

Reply:
Done in [`214fdeb` (#2808)](https://github.com/tardis-sn/tardis/pull/2808/commits/214fdebbd1d92a820c910f5a4b637a4c1f33dca4)